jobs/config.json

```diff
@@ -11234,72 +11234,6 @@
       "sig-scalability"
     ]
   },
-  "ci-kubernetes-kubemark-5-gce": {
-    "args": [
-      "--cluster=kubemark-5",
-      "--env-file=jobs/env/ci-kubernetes-e2e-kubemark-common.env",
-      "--extract=ci/latest",
-      "--gcp-node-image=gci",
-      "--gcp-node-size=n1-standard-2",
-      "--gcp-nodes=1",
-      "--gcp-project=k8s-jenkins-kubemark",
-      "--gcp-zone=us-central1-f",
-      "--kubemark",
-      "--kubemark-nodes=5",
-      "--provider=gce",
-      "--test=false",
-      "--test_args=--ginkgo.focus=\\[Feature:Empty\\] --gather-resource-usage=true --gather-metrics-at-teardown=true",
-      "--timeout=60m"
-    ],
-    "scenario": "kubernetes_e2e",
-    "sigOwners": [
-      "sig-scalability"
-    ]
-  },
-  "ci-kubernetes-kubemark-5-gce-last-release": {
-    "args": [
-      "--cluster=kubemark-latest",
-      "--env-file=jobs/env/ci-kubernetes-e2e-kubemark-common.env",
-      "--extract=ci/latest-1.9",
-      "--gcp-node-size=n1-standard-2",
-      "--gcp-nodes=1",
-      "--gcp-project=k8s-jenkins-kubemark",
-      "--gcp-zone=us-central1-f",
-      "--kubemark",
-      "--kubemark-nodes=5",
-      "--provider=gce",
-      "--test=false",
-      "--test_args=--ginkgo.focus=\\[Feature:Empty\\] --gather-resource-usage=true --gather-metrics-at-teardown=true",
-      "--timeout=60m"
-    ],
-    "scenario": "kubernetes_e2e",
-    "sigOwners": [
-      "sig-scalability"
-    ]
-  },
-  "ci-kubernetes-kubemark-5-prow-canary": {
```

Comment:
This was added back when we were discussing about test-infra changes breaking our jobs.

@krzyzacy - Is this still useful? In other words - do you watch the results of this job before updating prod prow image?

Reply:
My feeling is we may be able to get rid of this one.

Reply:
I'd prefer convert this to a kubemark-big-canary - we kinda want an equivalent canary job for each blocking presubmit, so the kubetest changes won't be surprising breaking changes.

Reply:
There already seems to be "ci-kubernetes-kubemark-100-canary" - so I think we can remove this.

Reply:
IMO that should be good enough to catch testinfra breakages of kubemark - wdyt?